aten/src/ATen/cpu/vec256/vec256_qint.h

```diff
@@ -206,7 +206,7 @@ struct Vec256<c10::qint8> {
   float_vec_return_type dequantize(
       Vec256<float> scale,
       Vec256<float> zero_point,
-      Vec256<float> scale_zp_premul) const {
+      Vec256<float> scale_neg_zp_premul) const {
```

Comment:
Any reason for the renaming?

Reply:
Yes, I spent at least 45 minutes debugging a numerical error because i forgot this should be `scale * -zp` instead of just `scale * zp`. This makes it more clear